apis/config/v1alpha1/clientconfig.go

```diff
@@ -155,55 +143,3 @@ func (c *ClientConfig) SetEditionSelector(edition EditionSelector) {
 	}
 	c.ClientOptions.CLI.UnstableVersionSelector = EditionStandard
 }
-
-// SetCompatibilityFile changes the compatibility file for the edition.
-func (c *ClientConfig) SetCompatibilityFile(edition EditionSelector) error {
```

Comment:
This function should not be part of the api definition and hence it was removed from here and moved to `pkg/v1/tkg/tkgctl/compatibility.go` as `SetCompatibilityFileBasedOnEdition`.